docs/Support/glossary.md

```diff
@@ -6,6 +6,10 @@ sidebar_position: 2
 
 ## Definitions of commonly used terms and core concepts.
 
+### **[Auth Sig](/SDK/Explanation/WalletSigs/authSig#authsigs)**
+In order to use Lit Protocol, you must present a wallet signature obtained from the user. This is referred to as an 'AuthSig' in the documentation.
+### **[Boneh-Lynn-Shacham (BLS) Signatures](https://medium.com/cryptoadvance/bls-signatures-better-than-schnorr-5a7fe30ea716)**
+A cryptographic algorithm that can be used for both signing and encryption.
```

Comment:
More explanation on BLS:

"BLS allows for signature aggregation and verification at scale using [Elliptic Curve cryptography](https://en.wikipedia.org/wiki/Elliptic-curve_cryptography).

Ethereum uses the BLS signature scheme to facilitate secure cryptography within the protocol. This method allows validators to sign messages, and these resulting signatures are then aggregated and verified at scale. This enables a Proof-of-Stake system with a large number of validators to function efficiently in production."

I took the above from here: https://docs.prylabs.network/docs/how-prysm-works/bls-cryptography
Which does a great job at explaining key terms.